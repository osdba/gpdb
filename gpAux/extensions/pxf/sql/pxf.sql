------------------------------------------------------------------
-- PXF read test
------------------------------------------------------------------

SELECT * from pxf_read_test order by a;
SELECT * from pxf_readcustom_test order by a;
