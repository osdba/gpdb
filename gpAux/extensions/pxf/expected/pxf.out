------------------------------------------------------------------
-- PXF read test
------------------------------------------------------------------
SELECT * from pxf_read_test order by a;
       a        |   b    |   c
----------------+--------+--------
 fragment1 row1 | value1 | value2
 fragment1 row2 | value1 | value2
 fragment2 row1 | value1 | value2
 fragment2 row2 | value1 | value2
 fragment3 row1 | value1 | value2
 fragment3 row2 | value1 | value2
(6 rows)

SELECT * from pxf_readcustom_test order by a;
       a        |   b    |   c
----------------+--------+--------
 fragment1 row1 | value1 | value2
 fragment1 row2 | value1 | value2
 fragment2 row1 | value1 | value2
 fragment2 row2 | value1 | value2
 fragment3 row1 | value1 | value2
 fragment3 row2 | value1 | value2
(6 rows)

